chemistry/amber/amberformat.py

```diff
@@ -814,15 +834,24 @@ def write_parm(self, name):
             for i in xrange(len(self.parm_data[self.charge_flag])):
                 self.parm_data[self.charge_flag][i] *= CHARGE_SCALE
 
+        if self.vmd_compat is not None:
+            self.delete_flag('CTITLE')
+
         # write version to top of prmtop file
         new_prm.write('%s\n' % self.version)
 
+        # then write pointers
+        self.flag_list.remove('POINTERS')
+        self.flag_list.insert(self.flag_list.index('TITLE')+1, 'POINTERS')
+
         # write data to prmtop file, inserting blank line if it's an empty field
         for flag in self.flag_list:
             new_prm.write('%%FLAG %s\n' % flag)
             # Insert any comments before the %FORMAT specifier
-            for comment in self.parm_comments[flag]:
-                new_prm.write('%%COMMENT %s\n' % comment)
+            # except VMD hates comments
+            if self.vmd_compat is None:
+              for comment in self.parm_comments[flag]:
+                  new_prm.write('%%COMMENT %s\n' % comment)
```

Comment:
Let's just have `convert_to_vmd_compat` delete all comments and get rid of the `vmd_compat` flag altogether.